### PR TITLE
IPROTO-283 Transcode bytes properties from JSON array

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/IPROTO-283

### CODE REVIEW

File: core/src/test/java/org/infinispan/protostream/ProtobufUtilTest.java [hunk 1]
```diff
@@ -226,6 +227,28 @@ public void testJsonWithDifferentFieldOrder() throws Exception {
       assertEquals(createAccount(), account);
    }
 
+   @Test
+   public void vectorFieldsTranscoding() throws Exception {
```
Comment (ryanemerson) [inline core/src/test/java/org/infinispan/protostream/ProtobufUtilTest.java:231]: `testJsonBytesWithIntArray`?

File: core/src/main/java/org/infinispan/protostream/impl/JsonUtils.java [hunk 2]
```diff
@@ -819,6 +840,7 @@ private static void expectField(String expectedFieldName, String actualFieldName
    }
 
    // todo [anistor] do we really need html escaping? so far I'm keeping it so we behave like previous implementation
+
```
Comment (ryanemerson) [inline core/src/main/java/org/infinispan/protostream/impl/JsonUtils.java:843]: Unnecessary whitespace

**Review by ryanemerson — COMMENTED** [review]
LGTM, just a couple of minor issues

File: core/src/test/java/org/infinispan/protostream/ProtobufUtilTest.java (hunk 1 above)
Comment (fax4ever) [reply core/src/test/java/org/infinispan/protostream/ProtobufUtilTest.java]: `testJsonBytesWithIntArray` is definitely a better name, thank you!

**Comment by fax4ever** [comment]
Thank you @ryanemerson for the review.
I think I've applied all the fixes here.

**Comment by fax4ever** [comment]
Hi @ryanemerson! Anything else to improve here or we can merge it? Thanks

**Comment by ryanemerson** [comment]
No all good @fax4ever, I just forgot about it!

**Comment by fax4ever** [comment]
Thank you @ryanemerson
